.clang-format

```diff
@@ -0,0 +1,6 @@
+BasedOnStyle: Google
```

Comment:
- [ ] It's necessary to fix the sorting order of the includes.

I propose:

1. `".*"`                            // relative includes
2. `<.*.hpp>`                   // our library private includes
3. `<yaclib\/.*>`               // our library public includes
4. `<[[:alpha:]_\/]+>`        // STL
5. `.*`                               // other libraries in our case: system headers

https://clang.llvm.org/docs/ClangFormatStyleOptions.html | IncludeCategories